assets/component-slideshow.css

```diff
@@ -43,9 +43,11 @@ slideshow-component .slideshow.banner {
   max-width: 54.5rem;
 }
 
-slideshow-component.page-width .slideshow__text {
-  border-right: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
-  border-left: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+@media screen and (max-width: 749px) {
+  slideshow-component.page-width .slideshow__text {
+    border-right: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+    border-left: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+  }
```

Comment:
Thanks for linking that article re: mobile first. I definitely see your point about this being an appropriate solution. I'll move this within the `max-width` query below to avoid having two instance of `max-width` close together.